conda/cli/conda_argparse.py

```diff
@@ -178,7 +178,7 @@ def do_call(args: argparse.Namespace, parser: ArgumentParser):
         # run the subcommand from executables; legacy path
         deprecated.topic(
             "23.3",
-            "24.3",
+            "25.3",
```

Comment:
Bumping removal of loading subcommands via executables

Critical projects have still not been properly updated to handle this deprecation (e.g., conda-smithy).

Reply:
Okay, I would suggest to open a ticket to track this work better, seems like we haven't made the outreach at least

Reply:
Added in https://github.com/conda/conda/pull/13634/commits/c186396b88036cd655c27aa6f861c91ea8b4e8d2